net/dino/Portfile

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           compiler_blacklist_versions 1.0
+PortGroup           github 1.0
+
+github.setup        dino dino 0.4.3 v
+revision            0
+categories          net www
+license             GPL-2
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+description         Modern XMPP (Jabber) chat client
+long_description    {*}${description} using GTK+/Vala.
+homepage            https://dino.im
+checksums           rmd160  4c19280920647390ea8a9de7b8e2a150bfb73341 \
+                    sha256  432d7c3b5170c595b1b31a8d64d73ded26e32af9f03a2d1a01828c22a8ade3fa \
+                    size    883755
+github.tarball_from archive
+
+# FIXME: on 10.6 it builds and launches, but trying to register an account results in an error:
+# (dino:74601): Pango-WARNING **: 23:01:36.818: Invalid UTF-8 string passed to pango_layout_set_text()
+
+depends_build-append \
+                    bin:cmake:cmake \
```

Comment:
why can this not use the "cmake" PortGroup?

Reply:
It uses a standard configure and handles CMake from there.

Reply:
okay, so upstream is a little confused about life 😉

Reply:
Yeah LOL